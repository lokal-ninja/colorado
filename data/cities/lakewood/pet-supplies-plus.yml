id: 7017801419
type: node
latitude: 39.69881
longitude: -105.1457
postcode: "80228"
city: Lakewood
street: West Alameda Parkway
housenumber: "13079"
phone: ""
opening_hours: ""
website: https://www.petsuppliesplus.com
