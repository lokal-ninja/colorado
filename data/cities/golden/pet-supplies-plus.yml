id: 13205349068
type: node
latitude: 39.73963
longitude: -105.19376
postcode: "80401"
city: Golden
street: South Golden Road
housenumber: "17205"
phone: +1-303-216-3294
opening_hours: Mo-Sa 09:00-21:00; Su 10:00-18:00
website: https://www.petsuppliesplus.com/store/co/golden/4017-golden/4017
